src/globalInfo/dxEnvVars.ts

```diff
@@ -0,0 +1,371 @@
+    synonymOf: DxWellKnownEnvVars.SF_API_VERSION,
+  },
+  [DxWellKnownEnvVars.SFDX_AUDIENCE_URL]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_AUDIENCE_URL)),
+    synonymOf: DxWellKnownEnvVars.SF_AUDIENCE_URL,
+  },
+  [DxWellKnownEnvVars.SFDX_CODE_COVERAGE_REQUIREMENT]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_CODE_COVERAGE_REQUIREMENT)),
+    synonymOf: DxWellKnownEnvVars.SF_CODE_COVERAGE_REQUIREMENT,
+  },
+  [DxWellKnownEnvVars.SFDX_CONTENT_TYPE]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_CONTENT_TYPE)),
+    synonymOf: DxWellKnownEnvVars.SF_CONTENT_TYPE,
+  },
+  [DxWellKnownEnvVars.SFDX_DEFAULTDEVHUBUSERNAME]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_DEFAULTDEVHUBUSERNAME)),
+    synonymOf: DxWellKnownEnvVars.SF_DEFAULTDEVHUBUSERNAME,
+  },
+  [DxWellKnownEnvVars.SFDX_DEFAULTUSERNAME]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_DEFAULTUSERNAME)),
+    synonymOf: DxWellKnownEnvVars.SF_DEFAULTUSERNAME,
+  },
+  [DxWellKnownEnvVars.SFDX_DISABLE_AUTOUPDATE]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_DISABLE_AUTOUPDATE)),
+    synonymOf: DxWellKnownEnvVars.SF_DISABLE_AUTOUPDATE,
+  },
+  [DxWellKnownEnvVars.SFDX_AUTOUPDATE_DISABLE]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_AUTOUPDATE_DISABLE)),
+    synonymOf: DxWellKnownEnvVars.SF_AUTOUPDATE_DISABLE,
+  },
+  [DxWellKnownEnvVars.SFDX_DISABLE_SOURCE_MEMBER_POLLING]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_DISABLE_SOURCE_MEMBER_POLLING)),
+    synonymOf: DxWellKnownEnvVars.SF_DISABLE_SOURCE_MEMBER_POLLING,
+  },
+  [DxWellKnownEnvVars.SFDX_DISABLE_TELEMETRY]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_DISABLE_TELEMETRY)),
+    synonymOf: DxWellKnownEnvVars.SF_DISABLE_TELEMETRY,
+  },
+  [DxWellKnownEnvVars.SFDX_DNS_TIMEOUT]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_DNS_TIMEOUT)),
+    synonymOf: DxWellKnownEnvVars.SF_DNS_TIMEOUT,
+  },
+  [DxWellKnownEnvVars.SFDX_DOMAIN_RETRY]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_DOMAIN_RETRY)),
+    synonymOf: DxWellKnownEnvVars.SF_DOMAIN_RETRY,
+  },
+  [DxWellKnownEnvVars.SFDX_IMPROVED_CODE_COVERAGE]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_IMPROVED_CODE_COVERAGE)),
+    synonymOf: DxWellKnownEnvVars.SF_IMPROVED_CODE_COVERAGE,
+  },
+  [DxWellKnownEnvVars.SFDX_INSTANCE_URL]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_INSTANCE_URL)),
+    synonymOf: DxWellKnownEnvVars.SF_INSTANCE_URL,
+  },
+  [DxWellKnownEnvVars.SFDX_JSON_TO_STDOUT]: {
+    description: messages.getMessage(camelCase(DxWellKnownEnvVars.SFDX_JSON_TO_STDOUT)),
+    synonymOf: DxWellKnownEnvVars.SF_JSON_TO_STDOUT,
```

Comment:
We should go through these and think carefully about whether it needs to map to `sf` or not. This one for example would have no effect on `sf`

Reply:
@mdonnalley The list of `SF` env vars is speculative at this moment, since I son't find any references to `SF` env vars in core. I will pull these out into a doc for further discussion.